cloud-config/prebuilt-config-yamls/20191112-pg11-u18-demo.yml

```diff
@@ -0,0 +1,112 @@
+#cloud-config
```

Comment:
They are important for development of deployment, and it's nice to have a backup in case the config build fails.  

We need the ones used to create the current amis, which could also be used as backups.
And it's nice to keep a copy of one before major updates.    

Ansible and a deployment repo would clean this up.